integrations-and-supported-tools/pytorch-lightning/notebooks/Neptune_PyTorch_Lightning.ipynb

```diff
@@ -55,7 +55,7 @@
    },
    "outputs": [],
    "source": [
-    "! pip install --quiet neptune-client>=0.11.0 pytorch-lightning>=1.5.0 scikit-plot==0.3.7 torchvision>=0.6"
+    "! pip install --quiet neptune-client>=0.11.0 git+https://github.com/Lightning-AI/lightning.git@neptune/installation#egg=lightning scikit-plot==0.3.7 torchvision>=0.6"
```

Comment:
`neptune/installation` branch doesn't exist within the `Lightning` repository. The correct pip command is 
`! pip install neptune-client>=0.11.0 git+https://github.com/Lightning-AI/lightning.git@39f899a80cb1118431576838cd3f86ead01404ff scikit-plot==0.3.7 torchvision>=0.6`

Steps to validate:
```python
! pip install neptune-client>=0.11.0 git+https://github.com/Lightning-AI/lightning.git@39f899a80cb1118431576838cd3f86ead01404ff 
! pip freeze | grep lightning

from pytorch_lightning import LightningModule, Trainer
from pytorch_lightning.loggers import NeptuneLogger

neptune_logger = NeptuneLogger(
    api_key="ANONYMOUS",
    project="common/pytorch-lightning-integration",
)
```

Reply:
`pytorch-lightling` should also be pinned to `1.6.4` in requirements.txt as well

Reply:
The idea would be to just temporarily adjust the notebook so that it works, and not touch anything else. @Herudaio mentioned that he got it to work with the line suggested by @Raalsky -- could you guys sort out which dependencies we should use?

Reply:
The branch doesn't exist as the fix was already merged to the master branch. Try to install `git+https://github.com/Lightning-AI/lightning.git@master#egg=lightning` (I didn't check that). + Fix should be released in a few days.

Reply:
I think we should stick to our practice of pinning external libraries instead of installing from git. Once the fix is released, we can pin it to the fixed version